sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/AdtDataType.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.DigitalTwins.Core.QueryBuilder
+{
+    /// <summary>
+    /// Data types in the ADT query language.
+    /// </summary>
+    public enum AdtDataType
+    {
+        /// <summary>
+        /// Boolean type in the ADT query language.
+        /// </summary>
+        AdtBool = 1,
```

Comment:
Let's not assign any explicit integer values to our enum unless we need the integer values.